pkg/cmd/cli/describe/deployments.go

```diff
@@ -276,66 +287,60 @@ func NewLatestDeploymentDescriber(client client.Interface, kclient kclient.Inter
 	}
 }
 
-func (d *LatestDeploymentDescriber) Describe(namespace, name string) (string, error) {
+func (d *LatestDeploymentsDescriber) Describe(namespace, name string) (string, error) {
 	config, err := d.client.getDeploymentConfig(namespace, name)
 	if err != nil {
 		return "", err
 	}
 
-	deploymentName := deployutil.LatestDeploymentNameForConfig(config)
-	deployment, err := d.client.getDeployment(config.Namespace, deploymentName)
-	if err != nil && !kerrors.IsNotFound(err) {
-		return "", err
+	var deployments []kapi.ReplicationController
+	if d.count == -1 || d.count > 1 {
+		list, err := d.client.listDeployments(namespace, labels.Everything())
+		if err != nil && !kerrors.IsNotFound(err) {
+			return "", err
+		}
+		deployments = list.Items
+	} else {
+		deploymentName := deployutil.LatestDeploymentNameForConfig(config)
+		deployment, err := d.client.getDeployment(config.Namespace, deploymentName)
+		if err != nil && !kerrors.IsNotFound(err) {
+			return "", err
+		}
+		if deployment != nil {
+			deployments = []kapi.ReplicationController{*deployment}
+		}
 	}
 
 	g := graph.New()
 	deploy := graph.DeploymentConfig(g, config)
-	if deployment != nil {
-		graph.JoinDeployments(deploy.(*graph.DeploymentConfigNode), []kapi.ReplicationController{*deployment})
+	if len(deployments) > 0 {
+		graph.JoinDeployments(deploy.(*graph.DeploymentConfigNode), deployments)
 	}
 
 	return tabbedString(func(out *tabwriter.Writer) error {
 		indent := "  "
-		fmt.Fprintf(out, "Latest deployment for %s/%s:\n", namespace, name)
-		printLines(out, indent, 1, d.describeDeployment(deploy.(*graph.DeploymentConfigNode))...)
+		node := deploy.(*graph.DeploymentConfigNode)
+		deploymentsCount := len(node.Deployments) + 1
```

Comment:
@fabianofranz I haven't yet digested all the graph stuff going on here, but I can't see yet why you'd need to add 1- there's no distinction between an "active" deployment and any other in the system, so if you list RCs filtered by the deploymentConfig label, you should get an accurate count...